src/plugins/data/common/field_formats/converters/bytes.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// @ts-ignore
+import numeral from '@elastic/numeral';
+// @ts-ignore
+import numeralLanguages from '@elastic/numeral/languages';
+import { KBN_FIELD_TYPES } from '../../kbn_field_types/types';
+import { FieldFormat } from '../field_format';
+import { TextContextTypeConvert } from '../types';
+
+const numeralInst = numeral();
+
+numeralLanguages.forEach(function(numeralLanguage: Record<string, any>) {
```

Comment:
We have duplicates of this code in 3 different formatters: bytes, percent, number.  
Can we reuse this code by creating one parent class for all of these types? In this case we also can simplify textConvert method because for all formatters above it looks very similar

Reply:
Done